src/nsupdate/main/models.py

```diff
@@ -253,8 +258,8 @@ def __str__(self):
         return u"%s.%s" % (self.name, self.domain.name)
 
     class Meta(object):
-        unique_together = (('name', 'domain'),)
-        index_together = (('name', 'domain'),)
+        constraints = [models.constraints.UniqueConstraint(fields=['name', 'domain'], name='unique_host_domain')]
+        indexes = [models.Index(fields=['name', 'domain'])]
         verbose_name = _('host')
         verbose_name_plural = _('hosts')
```

Comment:
something is wrong here. any idea, @nougad ?

*_together are deprecated in django 4.2 / soon.